docs/pages/api/telemetry/index.ts

```diff
@@ -0,0 +1,38 @@
+import { Analytics } from "@segment/analytics-node";
+import { NextApiRequest, NextApiResponse } from "next";
+
+export default async function forwardTelemetry(
```

Comment:
Do we want this to be an edge function? Genuinely not sure! (https://nextjs.org/docs/pages/building-your-application/routing/api-routes#edge-api-routes)

Reply:
i'm not sure either but it seems to work!